card/mod/02_basic_types/spec/set/type/pointer_spec.rb

```diff
@@ -52,11 +52,11 @@
     before do
       Card::Auth.as_bot do
         @card_name = "nonexistingcardmustnotexistthisistherule"
-        @pointer = Card.create :name=>"tp", :type=>"pointer", :content=>"[[#{@card_name}]]"
+        @pointer = Card.create name: "tp", type: "pointer", content: "[[#{@card_name}]]"
```

Comment:
Line is too long. [88/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.